ush/python/pygfs/task/aero_emissions.py

```diff
@@ -75,10 +105,261 @@ def execute(workdir: Union[str, os.PathLike], aprun_cmd: str) -> None:
         -------
         None
         """
+        AerosolEmissions.run(workdir)
+
+    @classmethod
+    @logit(logger)
+    def run(cls, workdir: Union[str, os.PathLike], current_date: str = None, forecast_dates: list = None, Config_dict: Dict = {}) -> None:
+        emistype = Config_dict.emistype
+        ratio = Config_dict.ratio
+        climfiles = sort(glob("{}{}".format(Config_dict.climfile_str, "*.nc")))
+        print(Config_dict.data_out['copy'][0][0])
+
+        if emistype.lower() == "qfed":
+            basefile = glob("qfed2.emis_*.nc4")
+        elif emistype.lower() == "gbbepx":
+            basefile = glob("GBBEPx_all01GRID.emissions_v*.nc")
+        elif emistype.lower() == "hfed":
+            basefile = glob("hfed.emis_*.x576_y361.*nc4")
+
+        dset = AerosolEmissions.make_fire_emission(
+            d=current_date,
+            climos=climfiles,
+            ratio=ratio,
+            scale_climo=True,
+            obsfile=basefile)
+        print(dset)
+
+        AerosolEmissions.write_ncf(dset, Config_dict.data_out['copy'][0][0])
+
+    @staticmethod
+    @logit(logger)
+    def open_qfed(fname) -> xr.Dataset:
+        """
+        Open QFED2 fire emissions data
```

Comment:
Open and do what with the data?
read variables, return the dataset of variables are found.  
What happens if none of the species are found?  Will it still return a dataset?  Should there be a warning/fatal error to trap such a condition?  
This should be done here, rather than some other method downstream expecting a dataset and finding empty dset.